### ISSUE TITLE
When are MONDO releases integrated with the MonarchInitiative Browser

### ISSUE THREAD

**Issue by larrybabb** [monarch-initiative/helpdesk#33]
It appears as if the last MONDO release available in the Monarch Initiative Browser is the [v2021-03-03](https://github.com/monarch-initiative/mondo/releases/tag/v2021-03-03) release. I did a very rudimentary test by looking for any of the new classes in the most recent releases and I could not find any until I hit the 3-30-21 new classes. The download data link from the monarchinitiative.org about box shows dates from the March and April 2021 timeframe.

Can you provide some insights into when the MONDO releases are made available in the monarchinitiative browser?
And can you provide some rationale as to why these would not be synchronized?

**Comment by nlharris** [comment]
Hi Larry, sorry for the delay in responding to your question. Perhaps @kshefchek can answer?

**Comment by matentzn** [comment]
Hey @larrybabb! Usually, we tie our ontology updates in the Monarch Knowledge Graph (which is the database underlying the Monarch Initiative Browser) to our data releases, which typically happen quarterly (this year 1 or 2 were skipped due to a major database migration we are working on). But based on your request here, we believe it makes sense to update the ontologies at least more regularly. We have been working on a way to update _just_ the ontologies to make it feasible to make more frequent updates to the browser, more in step with the ontology releases, and we are about halfway to that end. We will keep you updated.

Thank you very much for reaching out!

**Comment by nlharris** [comment]
Can this ticket be closed?

**Comment by matentzn** [comment]
No, we still have not figured out how to accelerate reloading ontologies independent of the rest of the data..

**Comment by matentzn** [comment]
This cant be closed but someone can move it to https://github.com/monarch-initiative/monarch-ingest/issues renaming it to "Refresh PHENIO on a weekly basis?"
